src/providers/LocalVideoProvider/docs/LocalVideoProvider.stories.mdx

```diff
@@ -23,6 +23,12 @@ To start/stop user's local video, use `toggleVideo` function from the `LocalVide
   // A function to set the value of `isVideoEnabled`
   setIsVideoEnabled: (isEnabled: boolean) => void;
 
+  // Whether or not video limit is reached, it defaults to false
+  isReachVideoLimit: boolean;
```

Comment:
The tense is not correct here, but I cant think of any good naming, maybe `hasReachedVideoLimit` or `isVideoLimitReached`?

Reply:
Since below you are listening to the `videoAvailabilityDidChange` event, may be we should give it a name related to video availability, and not mention video limit here.

Reply:
Changed it to `hasReachedVideoLimit` and `setHasReachedVideoLimit`